### DISCUSSION TITLE
Vicious is spelled wrong

### DISCUSSION THREAD

**Discussion by Pulparch** [bcholmes/StarTrek2d20#135] [General]
Hi, thank you for all your support of the tool, if you get a chance can you correct the spelling of viscous to vicious? When my players tried to modify their shuttle to have a torpedo it came up, not a huge deal.

**Comment by bcholmes** [comment]
Wait! Are you *sure* that the disruptors aren't supposed to be sticky/gooey?

**Comment by Pulparch** [comment]
Pretty sure. Loved the interview you gave with Continuing Missions/Studio
Tembo/Michael.
